packages/nhsuk-frontend/src/nhsuk/components/header/template.njk

```diff
@@ -54,13 +53,14 @@
   {{- nhsukAttributes(params.attributes) }} data-module="nhsuk-header">
   <div class="nhsuk-header__container nhsuk-width-container {%- if params.containerClasses %} {{ params.containerClasses }}{% endif %}">
     <div class="nhsuk-header__service">
+    {% set ariaLabel = params.logo.ariaLabel | default("NHS") %}
```

Comment:
Mind if we keep this inside the macro?

Makes it a tiny bit easier to accidentally output `alt=""` otherwise

Saves having to check the HTML output again for stray new lines too